button-merchant/src/main/java/com/usebutton/merchant/ButtonApi.java

```diff
@@ -47,4 +47,9 @@ PostInstallLink getPendingLink(String applicationId, String ifa,
     @WorkerThread
     Void postActivity(String applicationId, String sourceToken, String timestamp, Order order)
             throws ButtonNetworkException;
+
+    @Nullable
+    @WorkerThread
+    Void postOrder(Order order, String applicationId, String sourceToken,
```

Comment:
Why `postOrder` and not `reportOrder`?

Reply:
The naming convention throughout the entire ButtonApi class is `requestMethod` + `endpoint`  `reportOrder` is the public facing method